test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -761,3 +761,44 @@ Feature: Bind values from a secret referred in backing service resource
             """
             bla
             """
+
+    Scenario: Inject binding to an application from a Secret resource referred as service
```

Comment:
Given that it is expected the operator to recover from situations where the referred Secret comes to existence after the ServiceBinding resource is created, it'd be nice to see a reproduction of this scenario in the acceptance test suite.

Reply:
Done. 494077eab6dd88dd5b5db0a7060bf41b78d9fddf